_rules/inline-link-in-paragraph-has-distinguishable-style-not-based-on-color-alone-88407d.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: 88407d
-name: Inline link has distinguishable style or content not based on color alone
+name: Inline link has distinguishable style not based on color alone or content that indicates it is a link
```

Comment:
I feel it,s not fully clear that the "or" applies to "distinguishable style" (rather than to "based on color").
```suggestion
name: Inline link has either distinguishable style not based on color alone or content that indicates it is a link
```